bears/js/JSONFormatBear.py

```diff
@@ -1,13 +1,13 @@
 import json
 from collections import OrderedDict
 
+from coala_utils.param_conversion import negate
 from coalib.bearlib import deprecate_settings
 from coalib.bearlib.spacing.SpacingHelper import SpacingHelper
 from coalib.bears.LocalBear import LocalBear
 from coalib.misc.Compatibility import JSONDecodeError
 from coalib.results.Diff import Diff
 from coalib.results.Result import Result
```

Comment:
As it's inside JSONFormatBear only, I would add a tag to the commit message. As you would then exceed the 50 chars limit, you could shorten to
```
JSONFormatBear: Reorder imports
```
(btw: "to reorder" is a full verb, no need for "re-order" :D)